src/validator.js

```diff
@@ -0,0 +1,52 @@
+const { get, isNil } = require("lodash");
+const buildRoutingGraph = require("../src/helpers/build-routing-graph");
+const detectOrphan = require("./features/routing/detect-orphan");
+
+const failFast = errors => {
+  if (errors.errors.length > 0) {
+    throw new Error(JSON.stringify(errors, null, 2));
+  }
+};
+
+//eslint-disable-next-line
+const validatePage = (page, ctx, errors, options) => {
```

Comment:
What rule is being disabled here?

Reply:
There's no validation scripts yet that require an argument to be passed to the `options` parameter, so that isn't currently being used. Eslint is throwing a `no-unused-vars` error. 

I left in the `options` parameter to make it clear that's the designed behaviour, but it can be taken out if you think it's not necessary.  (Options could be used for toggling whether or not a test should use the real function or a mock function, for example.)

Reply:
tbh I'd probably just take it out, it's not being used and it's pretty easy to add back in when it needs to be used. Also when disabling a line it's really useful to say whats being disabled as well e.g. `//eslint-disable-next-line no-unused-vars` this also has the advantage of it only disabling that rule so if there are other issues they'll still be caught.